forge-gui/res/cardsfolder/upcoming/polygraph_orb.txt

```diff
@@ -0,0 +1,14 @@
+Name:Polygraph Orb
+ManaCost:4 B
+Types:Artifact
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDig | TriggerDescription$ When CARDNAME enters the battlefield, look at the top four cards of your library. Put two of them into your hand and the rest into your graveyard. You lose 2 life.
+SVar:TrigDig:DB$ Dig | DigNum$ 4 | DestinationZone2$ Graveyard | NoReveal$ True | ChangeNum$ 2 | SubAbility$ DBLoseLife
+SVar:DBLoseLife:DB$ LoseLife | LifeAmount$ 2
+A:AB$ Repeat | RepeatSubAbility$ RepeatTorment | RepeatPlayers$ Opponent | Cost$ 2 T CollectEvidence<3> | SpellDescription$ Each opponent loses 3 life unless they discard a card or sacrifice a creature. (To collect evidence 3, exile cards with total mana value 3 or greater from your graveyard.)
```

Comment:
AI was stuck in an infinite loop, don't know how

Reply:
you used wrong Effect

Reply:
but the real question is why are you even using Repeat when you're just iterating over them again below anyway?